eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubSnpNoHgvsToClusteredVariantProcessor.java

```diff
@@ -72,4 +80,48 @@ public DbsnpClusteredVariantEntity process(SubSnpNoHgvs subSnpNoHgvs) throws Exc
         return variantEntity;
     }
 
+    /**
+     * If the RefSnp type cannot be determined because none of the variants are type INS, DEL or INDEL, the type INDEL
+     * is assigned by default and the SubSnps would be declustered in
+     * {@link SubSnpNoHgvsToDbsnpVariantsWrapperProcessor }
+     */
+    private VariantType getVariantTypeWhenDiv(SubSnpNoHgvs subSnpNoHgvs) {
+        VariantType variantType = null;
+        List<String> alternateAlleles = subSnpNoHgvs.getAlternateAllelesInForwardStrand();
+        VariantType ssVariantType;
```

Comment:
It is not needed to track the value of this outside the loop, so its declaration can be moved inside it.